variant_remapping_tools/tests/test_replace_refs.py

```diff
@@ -0,0 +1,31 @@
+import os
+from unittest import TestCase
+
+from variant_remapping_tools.replace_refs import process_variant
+
+
+class TestReplaceRef(TestCase):
+
+    def test_process_variant(self):
+        # Reference allele from new genome was inserted in the vcf file and as a result the ref and alt are the same
+        line = '\t'.join(['chr1', '1', '.', 'G', 'G', '.', '.', '.'])
+        oldrefalleles_file = os.path.join(os.path.dirname(__file__), 'oldrefalleles.txt')
+        line_num = 0
+        # The old ref allele was reinserted
+        assert process_variant(line, line_num, oldrefalleles_file) == 'chr1	1	.	C	G	.	.	.'
+        # Note that I think this is wrong: the old ref allele should have been inserted in the ALT not in REF column
```

Comment:
We can discuss this in the subsequent issue ;-)

But just to shed some light on this, the rationale was like this. If the original file describes change from A to G, and in the new genome G is the reference allele, we don't want to flip the change in the VCF to “G to A”, because it's a completely different _change._ Especially if the old VCF had some functional annotations etc. attached to the change _from_ REF _to_ ALT, swapping them could have disastrous results.